### PR TITLE
Added (useful) error for using lists in filter by level

### PR DESCRIPTION
**Pull request by Rlamboll**
…rbidden column.

# Please confirm that this PR has done the following:

- [x] Tests Added
- [x] Documentation Added
- [x] Name of contributors Added to AUTHORS.rst
- [x] Description in RELEASE_NOTES.md Added

## Adding to RELEASE_NOTES.md (remove section after adding to RELEASE_NOTES.md)

Please add a single line in the release notes similar to the following:

```
- (#XX)[http://link-to-pr.com] Added feature which does something
```

# Description of PR

Please describe the changes introduced by this PR.

### CODE REVIEW

**Comment by Rlamboll** [comment]
@danielhuppmann The tests indicate a problem with the cached dependencies, I don't think this is related to my changes though?

**Comment by danielhuppmann** [comment]
> @danielhuppmann The tests indicate a problem with the cached dependencies, I don't think this is related to my changes though?

Fixed via #714

**Comment by danielhuppmann** [comment]
Closed and reopened to re-trigger tests after fixing the dependency issues

**Comment by danielhuppmann** [comment]
Still failing, maybe need to rebase?

File: RELEASE_NOTES.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ Bump minimum version of **pandas** to v1.2.0 to support automatic engine selecti
 
 ## Individual updates
 
+- [#713](https://github.com/IAMconsortium/pyam/pull/713) Informative error when using lists for filter by level. `Level' now a forbidden column.
```
Comment (danielhuppmann) [inline RELEASE_NOTES.md:10]:
```suggestion
- [#713](https://github.com/IAMconsortium/pyam/pull/713) Informative error when using lists for filter by level. `Level` now a forbidden column. 
```

**Comment by Rlamboll** [comment]
OK, looks like the rebase fixed it!

File: pyam/utils.py [hunk 2]
```diff
@@ -428,6 +428,11 @@ def find_depth(data, s="", level=None):
         whether depth satisfies the condition (equality if level is int,
         >= if ``.+``,  <= if ``.-``)
     """
+    if type(level) == list:
```
Comment (danielhuppmann) [inline pyam/utils.py:431]: Maybe better? From `utils.py`...
```suggestion
    if islistable(level):
```

File: pyam/utils.py [hunk 3]
```diff
@@ -428,6 +428,11 @@ def find_depth(data, s="", level=None):
...
+        raise ValueError(
+            "Level is only run with ints or strings, not lists. Use strings with "
+            "integers and + or - to filter by ranges."
```
Comment (danielhuppmann) [inline pyam/utils.py:434]:
```suggestion
            f"'level' must be int or str, found: {level}"
```

**Review by danielhuppmann — COMMENTED** [review]
Many thanks - two minor suggestions, then good to go!

**Review by danielhuppmann — APPROVED** [review]
Thanks @Rlamboll

**Pull request merged**